test_multiply_duplex AssertionError

```python
_________________________ TestEVA.test_multiply_duplex _________________________
[gw6] linux -- Python 3.8.5 /usr/bin/python3

self = <test_eva_protocol.TestEVA testMethod=test_multiply_duplex>

    async def test_multiply_duplex(self):
        data_set_count = 10
    
        self.overlay(0).eva_protocol.block_size = 10
        self.overlay(1).eva_protocol.block_size = 10
        self.overlay(2).eva_protocol.block_size = 10
    
        # create 10 different data sets for each direction (0->1, 0->2, 1->0, 1->2, 2->0, 2->1)
        participants = [
            (self.peer(0), self.overlay(0)),
            (self.peer(1), self.overlay(1)),
            (self.peer(2), self.overlay(2)),
        ]
    
        data = [
            (p, list((os.urandom(1), os.urandom(100), random.randrange(0, 256)) for _ in range(data_set_count)))
            for p in permutations(participants, 2)
        ]
    
        for ((_, community), (peer, _)), data_set in data:
            for d in data_set:
                community.eva_send_binary(peer, *d)
    
        await drain_loop(asyncio.get_event_loop())
    
        assert len(self.overlay(0).received_data) == 2
        assert len(self.overlay(1).received_data) == 2
        assert len(self.overlay(2).received_data) == 2
    
        data_sets_checked = 0
        for ((peer, _), (_, community)), data_set in data:
>           assert community.received_data[peer] == data_set
E           AssertionError: assert [(b'\xed', b'...b4', 97), ...] == [(b'\xed', b'...b4', 97), ...]
E             Right contains 3 more items, first extra item: (b'G', b'5\x84\x0e$\x9a\xd6\x05\xaba>\x1f\xfe\x12\x98ev\x1dE\xa87\xc3\xce\xc5)\xbcgw\x97\x13\xaet\xa1\xde:\xc3\x06\xea...9f\xba+`\x85\x1d,+\xf7v\x99h9\x07\x80\x87\xca\xfb\x13|S\xc5\xf2k"8R\xe9\xe4\xca%\x84z\xe3\xa6H\xf0\xe8c\x02o\x02/', 80)
E             Full diff:
E               [
E                (b'\xed',
E                 b'\x96\x80k\xd4k\x1f\xbb\xe4[%\x93#\xd4m\xe7\xbc\xd4\x11\xbc7\\\x10\x0f\x8a'
E                 b'=\x93\x98`\xc5\xf0*\x85$V*\x8bQ\x906\x905\xa6\x8b\x18\x99%M\xfd\xdfq\r\\'
E                 b'\xe7tQi\xbbU\xe3S\x0c\x83\xbb\xa4\xad\xce\xdcj\x90\xff7\x10*xY\x1f'
E                 b'\x8dG\x16\xe1!t\xff\x97S(\xeb\xe6\x8f=EP\x95$%\xd8Z\xcb\xff~',
E                 15),
E                (b'\x97',
E                 b'v:\xa4\xa5EZ%\xfc\xc5\xfcdS\x8b\x9d,\x89\x01\xb8 -?\xa2\xad8%\x12J\xc1'
E                 b'\xc3\xee\xbah\xa4\xba\xf30W.%\xf4f\xaf^\x88\x8f\xb4\x8d=\x03s&\xb5'
E                 b'\x0f\xe2\x86\xd5\x1c\xed\x16\x08\xea\xfa6\'"h\xe8\x1f\x1a)\x82\n'
E                 b'\xb2\xd8\x84(\x15\x10\x83I\xf9\xb0.\xf2a\xc3\xd71\xf1T\x19c&\x8d\xa5/'
E                 b't\x16\x97\x9f',
E                 48),
E                (b'\xb2',
E                 b'!\xfa\x96\xda1\x08\x04h|\x1f\xe1\xec\x19\x13UI\xd3a/\xde}\x93\xe3\x10'
E                 b'\x85\x1b\xef,\x9a\x0e\xf7w\xb7\x82RH\xb4\xd1\xc6\x0c\xf3\x88\x02S\x9dki\x19'
E                 b"O\x1d\xf0\x81[\xe59\xf4\x14x\xb3m\x9aE'\xc4Wo\xea=I\xcf\xddT\xc5\xc5EW"
E                 b'+J\x84t\xb2S\x13y\xec\xef[\xcc\xeaV\x0bL\xac\xec\xea[\xc4\x0f%\xda',
E                 151),
E                (b'=',
E                 b'(\x12=\xcc\xee\x82\xef\x1d\x14&iL<\xc9#\x16\xefuK\xdfH\xc4\r\x19\x9d\xeeB&'
E                 b',g\x0e[\xa6\xabH\x8eXk~\xcb\xfd\xf8?u\x87\xd27\x86w\x1a=x*#\xc5I\x1dd:\xae'
E                 b'\xcc\xd0?.\xa1\xb1\x8ex\xda\xc3\x86U\xa4M\\|\x9fG\xf0js\x88\x03K'
E                 b'\x0c1\xfe\xb3\x87\xce:`{\x8c\xa4\xe1$\xb5\xe8=',
E                 92),
E                (b'\x14',
E                 b'\xae\x99KCL\xbd \x84}\x83\x92\x01\x1bz<\xbd\x7f\xc5\xe1\xab\xedOh\xa0'
E                 b'a\x91\tx\xb4N\x19\xd6M^\xba\xe9\x1e\xbb\x91\xe4\xac\x02\x02\xd9\x18\x1e0>'
E                 b'\x87\x1b\xe9\xcbc\x91\xe1#%\x1b\x9e\xef\xd5\xb7\x83J\xdc\xd8&\x94\xb7_\x1b`'
E                 b'\xc9MW\xc2z.\xb7f2\x14\x9c\x80\xdc\x08G\xcety<\xe3\xeaa\xf4\x13'
E                 b'\xcb\xa3&\x9c',
E                 165),
E                (b'\xe6',
E                 b'O\xba\x11\xecT\x90P"Q\xf0TM\x0fdB\x10\x8a6H\x82o\xea\x93\x84\xb6\xb7\xee.'
E                 b"\x96\xb6'\x18_\x05\xe0AQ\xe5,\xf0n\xb0\x04\xd08\xe3\xca\x01aZ41"
E                 b'\xc8\x18\xc3\xd5\xbe\x91\x93\x86\xe2\xc9]%\xdbp\x1d\xeb\xd7\xa2C\x17'
E                 b'\xa0\x1c\xad\x1b\xdcVU\xb9d#$\x07\n\x84\xd3\xe5\x8c\xeck\x13\x90sw)'
E                 b'\xb1$$\xb4',
E                 97),
E                (b'\xc9',
E                 b'}U\x97\x03~d\xb7\x12\xa5\xfc\x93\xed/p!EsT\xe4\x10\xa1\x1c\xa7\xc8'
E                 b'\xc7.\xd1\x8em8\x16\x95\x15v\xcaZ\x15\xd3\x80\x07\x82\xf5%\xc0\xd7\xff~\x18'
E                 b'\x1d\xd2EIw:\x017,\x1bZ\xd7\xec\x92\xca\xd4\x8c\x8c\xc2\xe3\x93\xabs='
E                 b'|\x11"\xfa\xea\xdd\x89\xac\xdfH\x89\x8e@\x03z\x0eL\xc8\x93\xcb\x18M\x01\xd7'
E                 b'9\x8b\x9b\x03',
E                 63),
E             -  (b'G',
E             -   b'5\x84\x0e$\x9a\xd6\x05\xaba>\x1f\xfe\x12\x98ev\x1dE\xa87\xc3\xce\xc5)'
E             -   b'\xbcgw\x97\x13\xaet\xa1\xde:\xc3\x06\xea\x8f\xef\xff\xe0\x1d\xacl'
E             -   b'\xc2G\xc0\xe7\x1e\x01\x82\xa6\x1e\x07\x17$\xec\x9f\xba+`\x85\x1d,+\xf7v\x99'
E             -   b'h9\x07\x80\x87\xca\xfb\x13|S\xc5\xf2k"8R\xe9\xe4\xca%\x84z\xe3\xa6'
E             -   b'H\xf0\xe8c\x02o\x02/',
E             -   80),
E             -  (b'\xf9',
E             -   b'\x13\xc1\x8bn\xd1F\xe9\x0f\x19\x86\xf3\x9d\x98\xdd\x1d\xd2\xed\xeaJ\xdf'
E             -   b'\x1c>\xe1\t\x15\x8c\x07\x1d\xfa\xf1\xdf*\xebD#\xbb[}\xa3_PBS\x95'
E             -   b'\xe81\xad\xd2\xcf\x19\xc1)\xdcF\xe3\x03\x9a\xd5ab\x1b]\x07\xfe\x96\x089\xcf'
E             -   b'\xa9\xf3<\xef3\x99\xe6,z.\x86\xcc\xd9N\x9c\xb1\xa3\xdf\x97\xc3\xfa\x07$\xc2'
E             -   b'f\\\xab;\x16y\xc5\xbe',
E             -   44),
E             -  (b'\x11',
E             -   b'+\xb0\x96\xa8\xe6w)`\x08Rz\x00\x85\xf7\x95\xc6\xb7\\\xc6\xb8\xbeC\xb8\xfc'
E             -   b'\x1c-!\xc1\x7f\xc6np\xbbM\xd6?\xc5V\xf6\xb0\x03\xfe\xa15\xe8}\xfd\xc9'
E             -   b't\xd0v0>\x16\x8b\x12\xca\x82PY3\xc3PZ1<\x16lLZT#\xabHA\x8d\xfd\x93\xd8\x83'
E             -   b'1\x89\x9f!\xc9\x8c\xb5\x86"i0\\VfM\x9a\x1dir\xcd',
E             -   220),
E               ]

src/tribler-core/tribler_core/modules/metadata_store/community/tests/test_eva_protocol.py:306: AssertionError
------------------------------ Captured log call -------------------------------

```

Full details (too long for GitHub):
https://jenkins-ci.tribler.org/job/Test_tribler_main/362/consoleFull
https://jenkins-ci.tribler.org/job/Test_tribler_main/367/console

--- a/src/tribler-core/tribler_core/modules/metadata_store/community/tests/test_eva_protocol.py
+++ b/src/tribler-core/tribler_core/modules/metadata_store/community/tests/test_eva_protocol.py
@@ -275,6 +275,10 @@ class TestEVA(TestBase):
     async def test_multiply_duplex(self):
         data_set_count = 10
 
+        self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
+        self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False
+        self.overlay(2).eva_protocol.terminate_by_timeout_enabled = False
+
         self.overlay(0).eva_protocol.block_size = 10
         self.overlay(1).eva_protocol.block_size = 10
         self.overlay(2).eva_protocol.block_size = 10

